src/app/shared/theme-picker/theme-picker.ts

```diff
@@ -71,8 +71,8 @@ export class ThemePicker implements OnInit, OnDestroy {
 
   ngOnInit() {
     this._queryParamSubscription = this._activatedRoute.queryParamMap
-      .pipe(map(params => params.get('theme')), filter(Boolean))
-      .subscribe(themeName => this.installTheme(themeName));
+      .pipe(map((params: ParamMap) => params.get('theme')), filter(Boolean))
+      .subscribe((themeName: string) => this.installTheme(themeName));
```

Comment:
This is needed for `strictNullChecks`, but it wasn't detected previously. With the latest Angular/CLI/RxJS versions, it does become an error:
```ts
theme-picker.ts:75:51 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string'.
```